data/test/macros/unit_setup.cfg

```diff
@@ -40,3 +40,76 @@ no#endarg
         {OTHER}
     [/{ABIL}]
 #enddef
+
+#define TEST_ABILITY_NO_VALUE ABIL OTHER
+
+#arg ID
+1#endarg
+
+#arg SELF
+no#endarg
+
+#arg ALLIES
+no#endarg
+
+#arg ENEMIES
+no#endarg
+
+#arg CUMULATIVE
+no#endarg
+
+    [{ABIL}]
+        id = {ID}
+        name = _"test-{ABIL}_{ID}"
+        affect_self = {SELF}
+        affect_allies = {ALLIES}
+        affect_enemies = {ENEMIES}
+        cumulative = {CUMULATIVE}
+        {OTHER}
+    [/{ABIL}]
+#enddef
+
+#define VILLAGE_TUNNEL
+    {CUSTOM_TUNNEL (
```

Comment:
I think this should move down the file. While defining VILLAGE_TUNNEL above CUSTOM_TUNNEL works as long as they're both defined before the preprocessor sees a `{VILLAGE_TUNNEL ...}`, it seems more logical to define the generic one first.

Reply:
Updated.